[4.1.0] Add an Info note about removing QuickFix jar in MI (#5419)
wso2/docs-apim: en/docs/install-and-setup/setup/mi-setup/transport_configurations/configuring-transports.md

To enable the FIX transport listener and sender, set the following parameters to `true` in the deployment.toml file (stored in the `MI_HOME/conf` directory).

!!! Info

> 	Quickfix/J related jars are not shipped with the product by default. Please add them before enabling the FIX transport.

Reviewer: ```suggestion
	Quickfix/J related .jar files are not shipped with the product by default. You need to add them before enabling the FIX transport.
```
